react/features/toolbox/components/native/HangupMenu.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import React, { useCallback } from 'react';
+import { TouchableWithoutFeedback, View } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { createBreakoutRoomsEvent, createToolbarEvent, sendAnalytics } from '../../../analytics';
+import { appNavigate } from '../../../app/actions';
+import { ColorSchemeRegistry } from '../../../base/color-scheme';
+import { endConference } from '../../../base/conference';
+import { BottomSheet, hideDialog } from '../../../base/dialog';
```

Comment:
we recently separated sheets from dialogs, there is a hideSheet now.

Reply:
Fixed.